pages*: improve list placeholders and wording of descriptions (#12111)
tldr-pages/tldr: pages/linux/auracle.md

`auracle search '{{regular_expression}}'`

> - Display package information for one or more AUR packages:

Reviewer: ```suggestion
- Display package information about one or more AUR packages:
```

`auracle info {{package1 package2 ...}}`

> - Display the `PKGBUILD` file (build information) for a one or more AUR packages:

Reviewer: ```suggestion
- Display the `PKGBUILD` file (build information) of one or more AUR packages:
```
